os/kernel/semaphore/sem_timedwait.c

```diff
@@ -291,9 +291,12 @@ int sem_timedwait(FAR sem_t *sem, FAR const struct timespec *abstime)
 	leave_cancellation_point();
 
 	/* We are either returning success or an error detected by sem_wait()
-	 * or the timeout detected by sem_timeout().  The 'errno' value has
-	 * been set appropriately by sem_wait() or sem_timeout() in those
-	 * cases.
+	 * or the timeout detected by sem_timeout() or sem_trywait() called from
```

Comment:
These comments are for below set_errno and sem_wait affects.
In my opinoin, "We are either returning success or an error detected by sem_wait()" is enough at here.
Others should be at errout_disabled, not here.